tests/unit/test_migration.py

```diff
@@ -602,7 +602,7 @@ def test__migrate_document(tmp_path: Path):
         document_meta=document_meta, discourse=mocked_discourse, docs_path=tmp_path
     )
 
-    assert (file_path := (tmp_path / path)).is_file()
+    assert (file_path := tmp_path / path).is_file()
```

Comment:
Does using the walrus operator here, increase or decrease readability?
I feel the following is easier to read, but does take one more line of code.
```
file_path = tmp_path / path
assert file_path.is_file()
```

Reply:
I think it makes the code a tad bit more concise, but I do see your point. Shall we keep the current code as it and refactor it when it becomes complex enough that it would be better to make the `file_path` variable more explicit? :) Thank you for the feedback!